src/crawlers/crawler_all_artists.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import pandas as pd
+from bs4 import BeautifulSoup
+sys.path.append(os.path.abspath('../config'))
+sys.path.append(os.path.abspath('../crawler_utils'))
+from scraping_utils import ScrapingUtils as su
+from discography_utils import DiscographyUtils as du
+from common_utils import GenericUtils as gu
+
+root_link = 'https://www.azlyrics.com'
+div_tag = 'div'
+a_tag = 'a'
+div_attr = {'class': 'col-sm-6'}
+
+
+class AllArtists:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def scrape_all_artists():
+        artists = []
```

Comment:
We would be storing artists in a dictionary. We also need to store their raw_urls.
```python
# {artist_url: artist_name}
artists= {}
```